superset-frontend/src/SqlLab/components/LimitControl.tsx

```diff
@@ -99,18 +110,21 @@ export default class LimitControl extends React.PureComponent {
               value={textValue}
               placeholder={t(`Max: ${this.props.maxRow}`)}
               bsSize="small"
-              onChange={e => this.setState({ textValue: e.target.value })}
+              // @ts-ignore
```

Comment:
Why do we need the ts ignore here?

Reply:
Following the pattern in other files `src/explore/components/PropertiesModal.tsx`